chains/solana/contracts/tests/ccip/ccip_router_test.go

```diff
@@ -9424,6 +9426,176 @@ func TestCCIPRouter(t *testing.T) {
 				})
 			})
 
+			t.Run("When executing a report with oversized data, it fails, but it succeeds when buffered", func(t *testing.T) {
+				transmitter := getTransmitter()
+
+				msgAccounts := []solana.PublicKey{config.CcipLogicReceiver, config.ReceiverExternalExecutionConfigPDA, config.ReceiverTargetAccountPDA, solana.SystemProgramID}
+
+				sourceChainSelector := config.EvmChainSelector
+				message, _ := testutils.CreateNextMessage(ctx, solanaGoClient, t, msgAccounts)
```

Comment:
Out of curiosity, have you tested the program (either in integration tests or manually in devnet) with a PTT use-case?